docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
The problem with pushing this into the `dimensions`/`metricsSpec` is that it does not allow the schemaMapping to be different from the name. Ideally, I'd like a dimension named `foo` to map to something complex, like `hello.world[0].foo(value)`. By putting the mapping in the name, we lose the ability for these to be different. `metricsSpec` differentiates between `name` and `fieldName`, so it could be made to work there, but `dimensions` doesn't differentiate between the dimension name and the underlying fieldName. I think a consistent mechanism between the two makes sense, hence the addition of this new section rather than embedding it directly in `dimensions` and `metricsSpec`.